spinner.go

```diff
@@ -137,34 +148,14 @@ func (s *Spinner) Reverse() {
 
 // Color will set the struct field for the given color to be used
 func (s *Spinner) Color(c string) error {
+	var colorError error
```

Comment:
yeah, update to something along those lines. Early returns are pretty common with Go.